### PR TITLE
Update CHANGELOG.md table of contents, add SE-0287

### PR DESCRIPTION
**Pull request by Jumhyn**

### CODE REVIEW

**Comment by Jumhyn** [comment]
Aside: is there a reason (e.g., some sort of process change) that the changelog hasn't been updated in four months?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,26 +4,68 @@ CHANGELOG
...
+  let milkyChance: UIColor? = .init(named: "white")?.withAlphaComponent(0.5)
+  ```
+  
+  As is the case with the existing implicit member expression, resulting type of the chain must be the same as the (implicit) base, so it is not well-formed to write:
+  
+  ```swift
+  let cgMilky: UIColor = .white.withAlphaComponent(0.5).cgColor
+  ```
+  
+  (Unless, of course, approprite `white` and `withAlphaComponent` members were defined on `CGColor`.)
```
Comment (theblixguy) [inline CHANGELOG.md:47]: Typo:
```suggestion
  (Unless, of course, appropriate `white` and `withAlphaComponent` members were defined on `CGColor`.)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,26 +4,68 @@ CHANGELOG
...
+  
+  Members of a "chain" can be properties, method calls, subscript accesses, force unwraps, or optional chaining question marks. Furthermore, the type of each member along the chain is permitted to differ (again, as long as the base of the chain matches the resulting type)  meaning the following successfully typechecks:
```
Comment (theblixguy) [inline CHANGELOG.md:49]: Extra whitespace:
```suggestion
  Members of a "chain" can be properties, method calls, subscript accesses, force unwraps, or optional chaining question marks. Furthermore, the type of each member along the chain is permitted to differ (again, as long as the base of the chain matches the resulting type) meaning the following successfully typechecks:
```

**Comment by theblixguy** [comment]
> Aside: is there a reason (e.g., some sort of process change) that the changelog hasn't been updated in four months?

Not really, it's just that no one updated the changelog after each proposal acceptance. If you want, you can open a new PR to add the missing changeling entries!

File: CHANGELOG.md (hunk 1 above)
Comment (Jumhyn) [reply CHANGELOG.md:47]: 🤦 thanks!

**Comment by Jumhyn** [comment]
> If you want, you can open a follow-up PR to add the missing changelog entries!

Great, thanks @theblixguy! If I have the chance before someone else gets around to it I'll look more closely at what needs to be added—just wanted to make sure it wouldn't be wasted effort!

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,26 +4,68 @@ CHANGELOG
...
+
+  Implicit member expressions now support chains of member accesses, making the following valid:
+  
+  ```swift
+  let milky: UIColor = .white.withAlphaComponent(0.5)
+  let milky2: UIColor = .init(named: "white")!.withAlphaComponent(0.5)
+  let milkyChance: UIColor? = .init(named: "white")?.withAlphaComponent(0.5)
+  ```
+  
+  As is the case with the existing implicit member expression, resulting type of the chain must be the same as the (implicit) base, so it is not well-formed to write:
```
Comment (benrimmington) [inline CHANGELOG.md:41]:
```suggestion
  As is the case with the existing implicit member expression syntax, the resulting type of the chain must be the same as the (implicit) base, so it is not well-formed to write:
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,26 +4,68 @@ CHANGELOG
...
+  
+  ```swift
+  let cgMilky: UIColor = .white.withAlphaComponent(0.5).cgColor
```
Comment (benrimmington) [inline CHANGELOG.md:44]:
```suggestion
  let cgMilky: CGColor = .white.withAlphaComponent(0.5).cgColor
```

**Comment by Jumhyn** [comment]
Thank you @benrimmington!

**Comment by benrimmington** [comment]
@swift-ci Please smoke test

**Comment by benrimmington** [comment]
@swift-ci Please smoke test Linux platform

**Comment by Jumhyn** [comment]
@benrimmington Checks passed—can this get merged now?

**Comment by Jumhyn** [comment]
Thank you!

**Pull request merged**